SchedulerJobs/SchedulerJobs.Services/VideoApiService.cs

```diff
@@ -39,6 +51,24 @@ public async Task CloseConference(Guid conferenceId)
             response.EnsureSuccessStatusCode();
         }
 
+        public async Task<List<ClosedConferencesResponse>> GetClosedConferencesToClear()
```

Comment:
I think we should rename this to say something like GetClosedConferencesToClearIM. Otherwise, it is confusing with the other closed hearing methods.

Reply:
Yup will rename the method. I added the XML documentation for this method to be clear. see lines 22-25

Reply:
Thanks. Resolved.